packages/ui-components/src/__tests__/transactionStore.test.ts

```
@@ -458,66 +424,57 @@ describe('handleRemoveOrderTransaction', () => {
-		const mockTxHash = '0xremoveordertxhash';
-		(sendTransaction as Mock).mockResolvedValue(mockTxHash);
-
-		(waitForTransactionReceipt as Mock).mockResolvedValue({});
-
-		await handleRemoveOrderTransaction({
-			config: mockConfig,
-			orderbookAddress: mockOrderbookAddress,
-			removeOrderCalldata: mockRemoveOrderCalldata,
-			chainId: mockChainId,
-			subgraphUrl: ''
+		resolveIndexing!({
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)